pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -78,6 +78,8 @@ type Registry struct {
 	// effectively be pulled from "example.com/foo/bar/myimage:latest".
 	// If no Prefix is specified, it defaults to the specified location.
 	Prefix string `toml:"prefix"`
+	// The credential store for the registry
+	CredHelper string `toml:"cred_helper"`
```

Comment:
```suggestion
	CredentialHelper string `toml:"credential-helper"`
```
we use dashes elsewhere in this file already.

Also, document that the global default is used if `CredentialHelper` is not specified, and use `omitempty`.